### PR TITLE
AppCloud related orbit bundles.

### PR DESCRIPTION
**Pull request by gnudeep**
Please review and commit.

### CODE REVIEW

File: docker-client/1.0.10.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                            javax.net.ssl,
+                            javax.security.auth.x500,
+                            javax.validation;version="[1.1.0,2.0.0)",
+                            okio;version="[1.6.0,1.7.0)",
+                            org.apache.commons.codec.binary;version="[1.10.0,2.0.0)",
+                            org.apache.commons.compress.archivers;version="[1.9.0,2.0.0)",
+                            org.apache.commons.compress.archivers.tar;version="[1.9.0,2.0.0)",
+                            org.apache.commons.compress.compressors.bzip2;version="[1.9.0,2.0.0)",
+                            org.newsclub.net.unix;version="[2.0.0,2.1.0)",
+                            org.slf4j;version="[1.7.0,2.0.0)"
```
Comment (Nipuni) [inline docker-client/1.0.10.wso2v1/pom.xml]: Please add export version ranges

File: kubernetes-client/1.3.76.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                            com.mifmif.common.regex;resolution:=optional,
+                            com.squareup.okhttp;version="[2.7.2,3.0.0)",
+                            com.squareup.okhttp.logging;version="[2.7.2,3.0.0)",
+                            com.squareup.okhttp.ws;version="[2.7.2,3.0.0)",
+                            io.fabric8.openshift.client.dsl;resolution:=optional;version="[1.3.0,2.0.0)",
+                            io.fabric8.openshift.api.model;resolution:=optional;version="[1.0,2)",
+                            io.fabric8.openshift.client;resolution:=optional;version="[1.3.0,2.0.0)",
+                            io.fabric8.utils;resolution:=optional;version="[2.2.0,3.0.0)",
+                            io.fabric8.utils.ssl;resolution:=optional;version="[2.2.0,3.0.0)",
+                            javax.net,
```
Comment (Nipuni) [inline kubernetes-client/1.3.76.wso2v1/pom.xml]: What is the reason to have resolution:=optional here?

File: kubernetes-client/1.3.76.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                            javax.net.ssl,
+                            javax.security.auth.x500,
+                            javax.tools;resolution:=optional,
```
Comment (Nipuni) [inline kubernetes-client/1.3.76.wso2v1/pom.xml]: Please add export package versions

File: docker-client/1.0.10.wso2v1/pom.xml (hunk 1 above)
Comment (gnudeep) [inline docker-client/1.0.10.wso2v1/pom.xml]: Fixed in https://github.com/wso2/orbit/commit/3119064e24811ce44ddeb10f16d677a572e8aaa3

File: kubernetes-client/1.3.76.wso2v1/pom.xml (hunk 3 above)
Comment (gnudeep) [inline kubernetes-client/1.3.76.wso2v1/pom.xml]: fixed in https://github.com/wso2/orbit/commit/3119064e24811ce44ddeb10f16d677a572e8aaa3

File: kubernetes-client/1.3.76.wso2v1/pom.xml (hunk 2 above)
Comment (gnudeep) [inline kubernetes-client/1.3.76.wso2v1/pom.xml]: Fixed in https://github.com/wso2/orbit/pull/181/commits/56c63ed5002f7f98a4b5408f748c09e55ac58e65

**Pull request merged**